dbot/dbot/Models/User.cs

```diff
@@ -1,21 +1,28 @@
 ﻿using Discord;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 using System.Text;
 
 namespace dbot.Models
 {
     public class User
     {
-        public string Username;
-        public string Discriminator;
+       
+        public int ID { get; set; }

```

Comment:
Is ID required if we have Key?

Reply:
Actually i don't think so. the Users table should be able to work with a primary key of type string.  I can test and make this change.